tla-io/src/main/scala/at/forsyte/apalache/tla/imp/passes/SanyParserPassImpl.scala

```diff
@@ -58,10 +58,10 @@ class SanyParserPassImpl @Inject() (
     Right(modules.get(rootName).get)
   }
 
-  private def loadFromTlaString(content: String): PassResult = {
+  private def loadFromTlaString(content: String, aux: Seq[String]): PassResult = {
     val (rootName, modules) =
       new SanyImporter(sourceStore, annotationStore)
-        .loadFromSource(Source.fromString(content))
+        .loadFromSource(Source.fromString(content), aux.map(Source.fromString(_)))
```

Comment:
Why don't we pass the `String` directly? What's the benefit of wrapping it in a `scala.io.Source`?

Reply:
The preexisting method was written to take a source, and I was respecting that design design decision. The benefit of using a source, I assume, is that you can load the data incrementally when feasible, and don't need to read it all into memory.

Reply:
Right, in his case we already have a string. We can change the underlying method in `SanyParser` to only take a string if we think there's no reason to support loading from sources. I was trying to disturb the existing design as little as possible :)

Reply:
Also true 😆